pandas/tseries/tools.py

```diff
@@ -222,6 +222,11 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
         datetime strings, and if it can be inferred, switch to a faster
         method of parsing them. In some cases this can increase the parsing
         speed by ~5-10x.
+    origin : scalar convertible to Timestamp / string ('julian', 'unix'), default 'unix'
+        Define relative offset for the returned dates.
+        - If 'unix', offset is set to 1-1-1970
+        - If 'julian', unit must be 'D', and offset is set to beginning of Julian Calendar
+        - If Timestamp convertible, offset is set to Timestamp identified by origin
```

Comment:
are we requiring a normalized Timestamp? IOW, no hhmmss......? (I think yes, so ``pd.Timestamp(julian) == pd.Timestamp(julian).normalize()`` or its an error.